compiler/src/dotty/tools/dotc/interactive/InteractiveDriver.scala

```diff
@@ -132,7 +132,7 @@ class InteractiveDriver(val settings: List[String]) extends Driver {
 
     try {
       for {
-        entry <- zipFile.stream.iterator().asScala
+        entry <- zipFile.stream.toArray((size: Int) => new Array[ZipEntry](size))
         name = entry.getName
         tastySuffix <- tastySuffixes
         if name.endsWith(tastySuffix)
```

Comment:
While you're at, can you change the two lines above to:
```
tastySuffix <- tastySuffixes.find(name.endsWith)
```
I've mistakenly changed it in https://github.com/lampepfl/dotty/pull/3010/commits/0ae6f04b27d93a5e063420c9184062891ac8e53c